pleskdistup/actions/distupgrade.py

```diff
@@ -65,40 +65,49 @@ def estimate_revert_time(self) -> int:
         return 0
 
 
-class SetupDebianRepositories(action.ActiveAction):
+class SetupAptRepositories(action.ActiveAction):
     from_codename: str
     to_codename: str
-    name: str
-    plesk_sourcelist_path: str
+    sources_list_path: str
+    sources_list_d_path: str
+    _name: str
 
-    def __init__(self, from_codename: str, to_codename: str):
+    def __init__(
+        self,
+        from_codename: str,
+        to_codename: str,
+        sources_list_path: str = "/etc/apt/sources.list",
+        sources_list_d_path: str = "/etc/apt/sources.list.d/",
+        name: str = "set up APT repositories to upgrade from {self.from_codename!r} to {self.to_codename!r}",
+    ):
         self.from_codename = from_codename
         self.to_codename = to_codename
-        self.name = f"set up Debian repositories to upgrade from {from_codename!r} to {to_codename!r}"
-        self.sources_list_path = "/etc/apt/sources.list"
-        self.sources_list_d_path = "/etc/apt/sources.list.d/"
-        self.plesk_source_list_path = "/etc/apt/sources.list.d/plesk.list"
+        self.sources_list_path = sources_list_path
+        self.sources_list_d_path = sources_list_d_path
+
+        self._name = name
+
+    @property
+    def name(self):
+        return self._name.format(self=self)
 
     def _change_sources_codename(self, from_codename: str, to_codename: str) -> None:
```

Comment:
Proposal: Maybe we could move this function to some common -> deb specific module and add some unit tests

Reply:
Actually, the function looks too simple for separate testing. In future it should be replaced with something smarter, because simple string replacement doesn't seem too reliable.

Reply:
> In future it should be replaced with something smarter, because simple string replacement doesn't seem too reliable.

In this case I'm sure we should add some tests =)

Reply:
Anyway, it's just a suggestion. We could just add a github issue and leave it for now.